BuildaGitServer/GitServerPublic.swift

```diff
@@ -14,12 +14,14 @@ import Result
 
 public enum GitService: String {
     case GitHub = "github"
+    case EnterpriseGitHub = "enterprisegithub"
```

Comment:
With this addition, I think we should refactor GitService to the following

```
public enum GitService {
    case GitHub
    case EnterpriseGitHub(url: String)
    case BitBucket
}
```

Moving to an associated value only for the Enterprise case will make things much easier.